scripts/run-integ-tests.ps1

```diff
@@ -46,8 +46,10 @@ Invoke-Expression "${PSScriptRoot}\..\misc\netkitten\build.ps1"
 # Run the tests
 $cwd = (pwd).Path
 try {
-  cd "${PSScriptRoot}"
   $env:ECS_LOGLEVEL = 'debug'; go test -race -tags integration -timeout=40m -v ../agent/engine ../agent/stats ../agent/app
+  if (${LastExitCode} -ne 0) {
```

Comment:
Can we create a loop counter to repeat the tests?

Example-
```
$count =2
for(;$count -ne 0;$count--)
{
    if (${LastExitCode} -ne 0) {
        $env:ECS_LOGLEVEL = 'debug'; go test -race -tags integration -timeout=40m -v ../agent/engine ../agent/stats ../agent/app
    }
}
```

Reply:
> Can we create a loop counter to repeat the tests?
> 
> Example-
> 
> ```
> $count =2
> for(;$count -ne 0;$count--)
> {
>     if (${LastExitCode} -ne 0) {
>         $env:ECS_LOGLEVEL = 'debug'; go test -race -tags integration -timeout=40m -v ../agent/engine ../agent/stats ../agent/app
>     }
> }
> ```

Why do we need `$count =2` ? Shouldn't running this optionally one another time be sufficient?

Reply:
I have updated the code snippet above to break on the first try if the tests succeed. If not, the tests are executed one more time.